README.md

```diff
@@ -54,37 +54,37 @@ Where `<base-encoding-code-point>` is a code representing an entry in the multib
+U+0076,   v,        base32hex,          rfc4648 case-insensitive - no padding - highest char,           experimental
+U+0056,   V,        base32hexupper,     rfc4648 case-insensitive - no padding - highest char,           experimental
+U+0074,   t,        base32hexpad,       rfc4648 case-insensitive - with padding,                        experimental
+U+0054,   T,        base32hexpadupper,  rfc4648 case-insensitive - with padding,                        experimental
+U+0062,   b,        base32,             rfc4648 case-insensitive - no padding,                          final
+U+0042,   B,        base32upper,        rfc4648 case-insensitive - no padding,                          final
+U+0063,   c,        base32pad,          rfc4648 case-insensitive - with padding,                        draft
+U+0043,   C,        base32padupper,     rfc4648 case-insensitive - with padding,                        draft
+U+0068,   h,        base32z,            z-base-32 (used by Tahoe-LAFS),                                 draft
+U+006b,   k,        base36,             base36 [0-9a-z] case-insensitive - no padding,                  draft
+U+004b,   K,        base36upper,        base36 [0-9a-z] case-insensitive - no padding,                  draft
+U+007a,   z,        base58btc,          base58 bitcoin,                                                 final
+U+005a,   Z,        base58flickr,       base58 flicker,                                                 experimental
+U+006d,   m,        base64,             rfc4648 no padding,                                             final
+U+004d,   M,        base64pad,          rfc4648 with padding - MIME encoding,                           experimental
+U+0075,   u,        base64url,          rfc4648 no padding,                                             final
+U+0055,   U,        base64urlpad,       rfc4648 with padding,                                           final
+U+0070,   p,        proquint,           [PRO-QUINT],                                                    experimental
+U+1F680,  🚀,       base256emoji,       base256 with custom alphabet using variable-sized-codepoints,   experimental
 
 ```
 
-**NOTE:** Multibase-prefixes are encoding agnostic and their canonical form is a Unicode [code point], not an ASCII character or corresponding UTF-8 bytes.  
-Since UTF-8 is the most common context for binary data that gets prefixed as a multibase today, the UTF-8 column is provided as a reference for detecting multibase-prefixes, since most of these codes can be detected in the first byte in known-encoding contexts. 
+**NOTE:** Multibase-prefixes are encoding agnostic and their canonical form is a Unicode [code point], not the raw bytes. 
+The each character code point displays as in that encoding is provided for convenience, since most of these codes can be detected in the first byte in known-encoding contexts. 
```

Comment:
That sentence somehow got garbled and doesn't make sense anymore ;)

Perhaps rewriting it, that the "Character" column uses UTF-8 and is just there for convenience.